### PR TITLE
Us 755 create linear shunt compensator

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**

### CODE REVIEW

File: src/components/util/validation-functions.js [hunk 1]
```diff
@@ -6,6 +6,10 @@
  */
 
 export function validateField(value, toValidate) {
+    function toNumber(val) {
```
Comment (laurgarn) [inline src/components/util/validation-functions.js:9]: Yeah :-)

File: src/translations/network-modifications-en.json [hunk 2]
```diff
@@ -5,5 +5,7 @@
     "network_modifications/GENERATOR_CREATION": "Creating generator {computedLabel}",
     "network_modifications/LINE_CREATION": "Creating line {computedLabel}",
     "network_modifications/TWO_WINDINGS_TRANSFORMER_CREATION": "Creating 2 windings transformer {computedLabel}",
-    "network_modifications/BRANCH_STATUS": "Modification status line {computedLabel}"
+    "network_modifications/BRANCH_STATUS": "Modification status line {computedLabel}",
+    "network_modifications/SHUNT_COMPENSATOR_CREATION": "Creating shunt compensator {computedLabel}"
+
```
Comment (laurgarn) [inline src/translations/network-modifications-en.json:10]: Empty line that may disalign en and fr translations, no ?

**Review by laurgarn — APPROVED** [review]

**Review by niconoir — CHANGES_REQUESTED** [review]
Some logs to be removed

File: src/components/util/validation-functions.js [hunk 3]
```diff
@@ -29,12 +33,12 @@ export function validateField(value, toValidate) {
     if (toValidate.isValueLessOrEqualTo !== undefined) {
         if (value && toValidate.isValueLessOrEqualTo) {
             // TODO: remove replace when parsing behaviour will be made according to locale
-            const maxValue = Number(
-                toValidate.isValueLessOrEqualTo.replace(',', '.')
-            );
+            const maxValue = toNumber(toValidate.isValueLessOrEqualTo);
+            console.info('jbo max', maxValue);
```
[the remark this answers is not in the document]
Comment (JacquesBorsenberger) [reply src/components/util/validation-functions.js]: done

File: src/components/util/validation-functions.js [hunk 4]
```diff
@@ -29,12 +33,12 @@ export function validateField(value, toValidate) {
...
             if (!isNaN(maxValue)) {
+                console.info('jbo not nan ');
```
[the remark this answers is not in the document]
Comment (JacquesBorsenberger) [reply src/components/util/validation-functions.js]: done

File: src/components/dialogs/input-hooks.js [hunk 5]
```diff
@@ -0,0 +1,319 @@
...
+    clear,
+    voltageLevelOptions,
+    selectedNodeUuid,
+}) => {
+    const [connectivity, setConnectivity] = useState({
+        voltageLevel: null,
+        busOrBusbarSection: null,
+    });
+    const [errorVoltageLevel, setErrorVoltageLevel] = useState();
+    const [errorBusBarSection, setBusBarSection] = useState();
```
[the remark this answers is not in the document]
Comment (JacquesBorsenberger) [reply src/components/dialogs/input-hooks.js]: done

**Review by niconoir — APPROVED** [review]

**Pull request merged**